node-launchpad/.config/config.json5

```diff
@@ -20,11 +20,12 @@
       "<Ctrl-g>": {"HomeActions":"TriggerManageNodes"},
       "<Ctrl-G>": {"HomeActions":"TriggerManageNodes"},
       "<Ctrl-Shift-g>": {"HomeActions":"TriggerManageNodes"},
-      "<Shift-o>": {"HomeActions":"TriggerManageNodes"},
       "<h>": {"HomeActions":"TriggerHelp"},
       "<H>": {"HomeActions":"TriggerHelp"},
       "<Shift-h>": {"HomeActions":"TriggerHelp"},
-
+      "<Ctrl-r>": {"HomeActions":"TriggerResetNodesPopUp"},
+      "<Ctrl-R>": {"HomeActions":"TriggerResetNodesPopUp"},
+      "<Ctrl-Shift-r>": {"HomeActions":"TriggerResetNodesPopUp"},
```

Comment:
Three combinations?

Reply:
The `Shift-r` is just for a fall back. I'm not sure if it even works, i.e, some might think that `[Ctrl+R]` in the docs/footer might refer to capital `R`. So they'd try to perform ctrl + shift + r. It is just for that. It might just work without that, I haven't tested it.

Reply:
I have the same pattern for the other keybinds. 
It might be worth checking and cleaning that up later!